entries/m/maropost.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Maropost": {
```

Comment:
```suggestion
  "Maropost Marketing Cloud": {
```
They seem to have 2 different products, each with separate auth systems. 
![image](https://user-images.githubusercontent.com/20560218/126174691-d58bac46-5650-48e7-8661-dfb6c913a83a.png)
and email 2FA seems to only apply to the marketing cloud. The commerce cloud is hosted at a different domain and supports different 2FA methods (https://www.netohq.com/support/s/article/multi-factor-authentication)